cmd/tracee-rules/main.go

```diff
@@ -194,13 +195,22 @@ func listSigs(w io.Writer, sigs []types.Signature) error {
 }
 
 func listEvents(w io.Writer, sigs []types.Signature) {
-	var events []string
+	m := make(map[string]struct{})
 	for _, sig := range sigs {
 		es, _ := sig.GetSelectedEvents()
 		for _, e := range es {
-			events = append(events, e.Name)
+			if _, ok := m[e.Name]; !ok {
+				m[e.Name] = struct{}{}
+			}
 		}
 	}
+
+	var events []string
+	for k, _ := range m {
+		events = append(events, k)
+	}
+
+	sort.Slice(events, func(i, j int) bool { return events[i] < events[j] })
```

Comment:
I don't think this is needed any loner, as your map stores unique keys.

Reply:
Yes, just want to sort the keys to make the output more readable.

Reply:
yes, this is sorting not deduplicating @mtcherni95 . 
we should keep it not only for readability but also to make the output consistent. please keep it @vincent-pli